docs/modules/integrate/pages/legacy-cdc-connectors.adoc

```diff
@@ -0,0 +1,98 @@
+= Legacy CDC Connector
+
+Change Data Capture (CDC) refers to the process of observing changes
+made to a database and extracting them in a form usable by other
+systems, for the purposes of replication, analysis and many more.
+
+Change Data Capture is especially important to Hazelcast, because it allows
+for the _streaming of changes from databases_, which can be efficiently
+processed by the Jet engine.
+
+Implementation of CDC in Hazelcast {open-source-product-name} is based on
+link:https://debezium.io/[Debezium, window=_blank]. Hazelcast offers a generic Debezium source
+which can handle CDC events from link:https://debezium.io/documentation/reference/stable/connectors/index.html[any database supported by Debezium, window=_blank].
+However, we're also striving to make CDC sources first class citizens in Hazelcast,
+as we have done already for MySQL and PostgreSQL already are.
+
+== Installing the Connector
+
+This connector is included in the full distribution of Open Source Hazelcast.
+
+== CDC as a Source
+
+We have the following types of CDC sources:
+
+* link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/cdc/DebeziumCdcSources.html[DebeziumCdcSources, window=_blank]:
+  a generic source for all databases supported by Debezium
+* link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/cdc/mysql/MySqlCdcSources.html[MySqlCdcSources, window=_blank]:
+  a specific, first class Jet CDC source for MySQL databases (also based
+  on Debezium, but with the additional benefits provided by Hazelcast
+* link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/cdc/postgres/PostgresCdcSources.html[PostgresCdcSources, window=_blank]:
+  a specific, first class CDC source for PostgreSQL databases (also based
+  on Debezium, but with the additional benefits provided by Hazelcast
+
+To set up a streaming source of CDC data, define it using the following configuration:
+
+[source,java]
+----
+Pipeline pipeline = Pipeline.create();
+pipeline.readFrom(
+    MySqlCdcSources.mysql("customers")
+            .setDatabaseAddress("127.0.0.1")
+            .setDatabasePort(3306)
+            .setDatabaseUser("debezium")
+            .setDatabasePassword("dbz")
+            .setClusterName("dbserver1")
+            .setDatabaseWhitelist("inventory")
+            .setTableWhitelist("inventory.customers")
+            .build())
+    .withNativeTimestamps(0)
+    .writeTo(Sinks.logger());
+----
+
+For an example of how to use CDC data see xref:pipelines:cdc.adoc[our tutorial].
+
+=== Fault Tolerance
+
+CDC sources offer at least-once processing guarantees. The source
+periodically saves the database write ahead log offset for which it had
+dispatched events and in case of a failure/restart it will replay all
+events since the last successfully saved offset.
```

Comment:
```suggestion
CDC sources offer _at least once_ processing guarantees. The source
periodically saves the database write ahead log offset for which it has
dispatched events and in case of a failure/restart it will replay all
events since the last successfully saved offset.
```